doc/plugin/vifm-plugin.txt

```diff
@@ -108,6 +108,10 @@ opening directories using vifm.  Defaults to false.
 A string variable.  The command to use to open folders when
 g:vifm_replace_netrw is enabled.  Defaults to "Vifm".
 
+                                               *g:vifm_drop_gone_buffers*
+A boolean variable.  When true, upon exiting Vifm, previously open buffers
```

Comment:
This needs to say that only unmodified buffers get deleted, makes a difference.

Reply:
Thanks, should be fixed now.